articles/decoding-jwt-structure.md

```diff
@@ -0,0 +1,276 @@
+# Decoding JWTs Structure
+
+By Rishi Raj Jain
+
+<insert-fig>
+
+JSON Web Tokens (JWTs) are compact, URL-safe tokens commonly used in modern authentication and authorization systems, particularly with OAuth2 flows. This guide provides a technical deep-dive into JWT architecture, examining each component and its role in the token's security and functionality.
+
+JWTs are sometimes incorrectly referred to as "JWT Tokens" - this would expand to "JSON Web Token Token," which is redundant. We'll use "JWT" throughout this guide. Additionally, while JWTs are frequently used for authentication, they're technically just base64-encoded JSON containers that can serve various purposes beyond auth. This guide focuses on their authentication use case while explaining the underlying structure.
```

Comment:
Remove `JWTs are sometimes incorrectly referred to as "JWT Tokens" - this would expand to "JSON Web Token Token," which is redundant. We'll use "JWT" throughout this guide. Additionally, while `